samples/bluetooth/mesh/light/sample.yaml

```diff
@@ -17,3 +17,11 @@ tests:
       nrf5340dk_nrf5340_cpuapp_ns thingy53_nrf5340_cpuapp thingy53_nrf5340_cpuapp_ns
       nrf21540dk_nrf52840 nrf52833dk_nrf52833
     tags: bluetooth ci_build
+  sample.bluetooth.mesh.light.dfu:
+    build_only: true
+    integration_platforms:
+      - nrf52840dk_nrf52840
+      - nrf21540dk_nrf52840
```

Comment:
This comment is for both files: Is there any point in building for anything else then nrf52840dk_nrf52840? (And if needed, should it be nrf5340?)

Reply:
I added platforms that have been referenced as the target platforms in README.rst for features that overlay adds. 
I guess if we claim in the documentation platform list we have to check at least the sample is built for them